_posts/zheinsohn/2021-12-9-final-project-zheinsohn.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+author: zheinsohn
+title: zheinsohn final project reflection
+
```

Comment:
Could you close out this header by adding `---` after line 4? Otherwise, this looks good!